images/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div class="header_logo">My Logo</div>
+            <div class="bar">
+                <a href="#"><i class="fa fa-bars"></i></a>
+            </div>
+        </header>
+        <div class="main-container">
+        <section class="head-bar">
+            <article>
+                <div class="intro">
+                    <h1>I'm Girma Gald to see you! </h1>  
+                </div>
+                <div class="discription">
+                    <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! if you like what you  see and have a project you need coded, don't hestiate to contact me</p>
+                </div>
+                <div class="let-connect">
+                    <div class="connect-l">LET'S CONNECT</div>
+                    <div class="icon-i">
+                        <ul>
+                            <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                            <li><a href="#"><i class="fa fa-linkedin"></i></a></i></li>
+                            <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                            <li><a href="#"><i class="fa fa-github"></i></a></li>
+                            <li><a href="#"><i class="fa fa-hand-peace-o"></i></a></i></li>
+                        </ul>
+                    </div>
```

Comment:
- [ ] You did hard work on the project so I am giving you an optional suggestion but please implement it. Your social icons are totally different from Figma design so I would suggest you download 1 by one from Figma design and use it.
### your design
![hiss](https://user-images.githubusercontent.com/81727988/191897463-a9924859-8eec-463d-a18f-72aa045fac39.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191897477-c729bd74-8cf4-454f-93ae-d68ca30132af.png)